torch/__init__.py

```diff
@@ -300,10 +300,69 @@ def set_default_dtype(d):
     _C._set_default_dtype(d)
 
 def set_deterministic(d):
-    r"""Sets a global flag to force all operations to use a deterministic
-    implementation if available. If an operation that does not have a
-    deterministic implementation is called while this setting is True, the
-    operation will throw a RuntimeError.
+    r"""Sets a global flag to force normally nondeterministic operations to
+    use a deterministic implementation, if available. Or if any of these
+    operations do not have a deterministic implementation, they will throw
+    a :class:`RuntimeError`.
+
+    .. warning::
+        This feature is a work in progress, so it does not affect every
+        nondeterministic operation yet. All affected operations are listed below.
```

Comment:
Instead of the last two sentences, maybe just, "The following operations are affected by this flag:"

Yes there may be other operations affected because they make use of this list, but those should be reported as documentation bugs anyway, right?

Reply:
Makes sense, I've changed it.